web/dashboards/garage/js/garage.js

```diff
@@ -1,327 +1,331 @@
-      if (httpRequest.status === 200) {
-        return callback(JSON.parse(httpRequest.responseText));
-      } else {
-        console.log("Error", httpRequest.status, "on GET", url);
-        return callback(null);
-      }
-    }
-  };
-  httpRequest.open("GET", url);
-  httpRequest.setRequestHeader("Accept", "application/json");
-  httpRequest.send();
-}
-
-// Updare Time
-function updateClock() {
-  let now = new Date();
-  dateNode.innerText = now.toLocaleTimeString("en-US",{ hour12:false });
-}
-
-// Parse the config and start periodic data update 
-function handleConfigAndStart(response) {
-  if(response) {
-    config = response;
-    initialiseBeaver(config.hlcServerUrl);
-    updateGraphs();
-    setInterval(updateGraphs, config.updateGraphsMilliseconds);
-  } 
-  else {
-    console.log("Unable to get config.  Refresh page to try again.");
-  }
-}
-
-// The following code runs on startup...
-getJson(baseUrl + CONFIG_ROUTE, handleConfigAndStart);
-setInterval(updateClock, 1000);
+/**
+ * Copyright reelyActive 2019
+ * We believe in an open Internet of Things
+ */
+
+// Constants
+const BASE_ROUTE = "/api/garage";
+const CONFIG_ROUTE = BASE_ROUTE + "/config";
+
+// DOM elementsa
+let temperature = document.querySelector("#temperature")
+let humidity = document.querySelector("#humidity");
+let occupancyCount = document.getElementById("occupancycount");
+let dateNode = document.getElementById("date");
+let presenceLab = [];
+let presenceArray = [];
+let presenceOffice = [];
+let presenceReception = [];
+let managerList = [ "ac233fa152c2", "ac233fa152bb", "ac233fa152a7", "ac233fa152ae", "ac233fa152cc", "ac233fa152a9"];
+let internList = ["ac233fa152ac", "ac233fa152b7", "ac233fa152b5", "ac233fa152b1", "ac233fa152b0", "ac233fa152a8"];
+let visitorList = ["ac233fa152a5", "ac233fa152aa", "ac233fa152bd", "ac233fa152ba", "ac233fa152a6", "ac233fa152b2","ac233fa152b8","ac233fa152bc","ac233fa152e8"];
+let lab = document.querySelector('#lab');
+let office = document.querySelector('#office');
+let reception = document.querySelector('#reception');
+
+let visitor = document.querySelector("#visitors");
+let intern = document.querySelector("#interns");
+let manager = document.querySelector("#managers");
+let visitorReception = document.querySelector("#visitorsReception");
+let internReception = document.querySelector("#internsReception");
+let managerReception = document.querySelector("#managersReception");
+let visitorLab = document.querySelector("#visitorsLab");
+let internLab= document.querySelector("#internsLab");
+let managerLab = document.querySelector("#managersLab");
+
+//garage receivers
+let officeSensors = ["001bc5094081017a","001bc5094081017b","001bc50940810175", "001bc50940810174"];
+let receptionSensors = ["001bc50940810176","001bc5094081017d","001bc5094081017d","001bc50940810181"];
+let labSensors = ["001bc5094081017f","001bc50940810180","001bc5094081017e","001bc50940810177"];
+
+let storyMap = {

```

Comment:
these IDs are also in the lib folder, Connecting json-silo and chickadee to this will help us create them dymanically